wpilibNewCommands/src/main/native/cpp/frc2/command/Command.cpp

```diff
@@ -21,13 +21,6 @@ Command::~Command() {
   CommandScheduler::GetInstance().Cancel(this);
 }
 
-Command::Command(const Command& rhs) = default;
-
-Command& Command::operator=(const Command& rhs) {
-  m_isGrouped = false;
```

Comment:
Removing this is a functional change, right?

Reply:
Since m_isGrouped is the only member variable and there's no base class, the default assignment operator would effectively be this function, right?